### ISSUE TITLE
[Arcs] minor technicality for Decided Council

### ISSUE THREAD

**Issue by turmoilbyrd** [LederCards/cards#85]
Technically, [Armistice](https://cards.ledergames.com/card/ARCS-F216)'s _When Secured_ effect only place the agents. It doesn't explicitly say influence, so it could bypass the restriction on [Decided Council](https://cards.ledergames.com/card/ARCS-AID01B). Consensus is this shouldn't be allowed. So Decided Council might need an errata to say "agents cannot be placed on this card".

**Comment by jdyearsley** [comment]
Agreed, although Decided Council is hurting for space unless we take the graphic off. I wonder whether the better errata is on Armistice to make it into an influence action that specially lets you place the enemy agents. I'm currently not seeing any cards that have influence modifiers that would be affected by this change.

**Comment by jdyearsley** [comment]
Proposed errata: "Take all agents from here and the card it’s attached to. You must influence but using all these agents."

**Comment by emptierset** [comment]
My understanding is that this would make Psionic Liaisons trigger (a lot, potentially) when you copy to secure Armistice, but maybe you were looking for more destructive modifier interactions.

**Comment by turmoilbyrd** [comment]
Good point but it's not the first time we see Psionic Liaisons triggering a lot. Dealmakers can make it happen far more easily.

**Comment by turmoilbyrd** [comment]
> Proposed errata: "You must take all agents from here and the card it’s attached to, and influence using these agents."

This proposed errata didn't actually make it to the website? Do we still think it's appropriate?
